llrt_modules/src/modules/process.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+use std::env;
+
+// This is not the full module for now since we are not yet sure
+// how to implement the TIME_ORIGIN Atomic in what are supposed to
+// be independant modules.
```

Comment:
If you have suggestions on how to do that @richarddavison I am all hears.
It is very much tied to the VM, which is a core concept that we don't want in the modules.

Reply:
Other than vm.rs, TIME_ORIGIN is used for processes and performance. I believe that `process` is a Node.js API, so it is eligible to be moved to `llrt_modules`, but `performance` is a WinterCG API, so it will remain in `llrt_core` (although perf_hooks API may be implemented in the future...).

So we think it is appropriate to move the following instruction to `llrt_utils/src/lib.rs`. What do you think?

```rust
pub static TIME_ORIGIN: AtomicUsize = AtomicUsize::new(0);
```

Reply:
Probably but the problem really is that its not obvious for a user of the modules that it needs to set that thing.
Thats why I am saying it should done at module declaration probably?

Reply:
Hum. Can we move it to utils with a lazy init once_cell or similar? Then call the init from all models where it used? This will effectively only call it once with negligible overhead

Reply:
Probably not since we need the time the "VM" started. So there needs to be an explicit call from the user. 
The more I think about it, the more I feel this is something we should put in `rquickjs`.

Reply:
Right I see what you mean. It doesn’t have to tho. It’s just relative time. So it’s fine to init lazy. In LLRT runtime we should just run the init ASAP in the VM by getting the value

Reply:
Good idea, I will move it once this gets merged.